assets/js/layouts/playground.js

```diff
@@ -43,14 +48,21 @@ export class Playground {
 		this.$el.innerHTML = doc.querySelector(".playground").innerHTML;
 
 		// fade in the image once loaded
-		this.$el
-			.querySelector(".playground-header-figure img")
+		this.currentImg()
 			.addEventListener("load", function () {
```

Comment:
`Suggestion:`

```suggestion
		this.image.addEventListener("load", function () {
```

Depends on suggested change to getters, see below.